experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/object.spec.ts

```diff
@@ -38,9 +38,9 @@ function testObjectLike(testCases: TestCase[]) {
 	describe("Object-like", () => {
 		describe("satisfies 'deepEquals'", () => {
 			for (const { schema, initialTree } of testCases) {
-				const view = createTreeView(schema, initialTree);
+				const view = createTreeView2(schema, initialTree as NewFieldContent);
```

Comment:
why is the casting necessary?

Reply:
the old API used type any and the new one uses strong types. This generic code doesn't have enough type information for each member of the array to work without a cast, its just now much more explicit and safer.